dojo/templates/dojo/new_product_type.html

```diff
@@ -1,4 +1,16 @@
 {% extends "base.html" %}
+{% load static from staticfiles %}
```

Comment:
these should all be replaced with {% load static %} as staticfiles will be removed in django3

Reply:
Done